pkgs/applications/editors/geany/default.nix

```diff
@@ -47,7 +47,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "http://www.geany.org/";
     license = "GPL";
-    maintainers = [ stdenv.lib.maintainers.bbenoist ];
```

Comment:
I feel it is better to leave the original maintainer in place and let them remove themselves. What do you think?